alf/config_helpers.py

```diff
@@ -106,6 +107,8 @@ def get_env():
     global _env
     if _env is None:
         trainer_config = TrainerConfig(root_dir='')
+        trainer_config.random_seed = set_random_seed(
+            trainer_config.random_seed)
```

Comment:
If the random seed is set here, we need to guarantee that no other env is created before the first call of get_env(). This might be problematic. I think a better way is still call set_random_seed at the entry of the main function.

Reply:
moved to entry